volcli/commands.go

```diff
@@ -34,14 +30,15 @@ var Commands = []cli.Command{
 		Subcommands: []cli.Command{
 			{
 				Name:        "upload",
+				Flags:       GlobalFlags,
```

Comment:
@unclejack , this should not be needed I think as user can just do `volcli --volmaster=<> upload < ...`